### PR TITLE
Fixed to always encode

### PR DESCRIPTION
**Pull request by wmbest2**
Update README

### CODE REVIEW

**Comment by designatednerd** [comment]
README LGTM, @vokal-isaac How's this look vs. VOKMockUrlProtocol's handling of encoding?

File: README.md [hunk 1]
```diff
@@ -32,3 +32,26 @@ Use the Client
             .setClient(new MockClient(getContext(), "mocks"))
             .build();
 ```
+
+Notes
+----
+If you are unsure what to name your mocks, check the logs.  It will include both the expected name and the hash incase the filename is too long for your file system.
```
Comment (vokal-isaac) [inline README.md]: "in case" is two words.

**Comment by vokal-isaac** [comment]
Are you alphabetizing parameters outside of the beconding process?

File: README.md (hunk 1 above)
Comment (wmbest2) [inline README.md]: heh, I also it should be "Because of this*" down below.  Definitely tired when I wrote this.

**Comment by wmbest2** [comment]
I suppose so.  Because of the way the BencodingOutputStream handles SortedMap objects, it wont try to sort it twice.  So in this case, I am just doing extra work that probably would be done automatically by the library. 

I should be able to convert these all to regular Map<String, String> objects and get the same result.

File: README.md (hunk 1 above)
Comment (wmbest2) [inline README.md]: Based on that last comment, I apparently cannot form correctly formatted sentences.

Comment (designatednerd) [inline README.md]: wat you mean? you english good.

**Comment by designatednerd** [comment]
@wmbest2 Do you want to merge this and then make that update in another pull, or would you prefer to leave this pull open until that is resolved?

**Comment by wmbest2** [comment]
Go ahead and merge and make the change. Thank you!!

**Comment by designatednerd** [comment]
@wmbest2 Heh, didn't mean to imply that I'd make the change as I'm not 100% sure how this stuff works under the hood. Would you still like me to merge this? :smile:

**Comment by wmbest2** [comment]
Ahh sorry I thought you were referring to the typo which I think I never
fixed. The other thing is basically a non issue for now.

**Comment by designatednerd** [comment]
Oh! Yes, that, I am entirely capable of fixing.

**Pull request merged**